Sources/funnet/Combine/Refresh.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Refresh.swift
+//  FunNet
+//
+//  Created by Elliot Schrock on 6/12/22.
+//
+
+import Combine
+
+public extension CombineNetCall {
```

Comment:
Ditto here, maybe we wanna consider a separate podspec that can go back to the minimum version that we have in Core, but then stick a `@available` in this extension